src/effects/VST/VSTEffect.cpp

```diff
@@ -1631,20 +1636,20 @@ bool VSTEffect::Load()
       wxLogNull nolog;
 
       // Try to load the library
-      auto lib = std::make_unique<wxDynamicLibrary>(realPath);
-      if (!lib) 
+      auto lib = std::make_unique<wxDynamicLibrary>(mRealPath);
+      if (!lib)
          return false;
 
       // Bail if it wasn't successful
       if (!lib->IsLoaded())
          return false;
 
       // Try to find the entry point, while suppressing error messages
-      pluginMain = (vstPluginMain) lib->GetSymbol(wxT("VSTPluginMain"));
-      if (pluginMain == NULL)
+      mPluginMain = (vstPluginMain)lib->GetSymbol(wxT("VSTPluginMain"));
```

Comment:
Mac build also requires `mPluginMain` in lines above

Reply:
Linux compilation will also need a fix in the second commit.

Reply:
right, done